src/services/github.service.ts

```diff
@@ -0,0 +1,56 @@
+import {SearchGitHubUsersRequest,GetGitHubRepoLanguageRequest} from '../interfaces/github.client';
+
+
+
+import { GetGitHubRepoLanguageResponse, GitHubGetUsersResponse } from '../interfaces/github.response';
+
+
+
+export const searchGitHubUsers = async (request: SearchGitHubUsersRequest): Promise<GitHubGetUsersResponse> => {
```

Comment:
Please do not move github.client functions here, instead you should call the functions from here.